### PR TITLE
Fix version numbers associated with RISCV Zc* extensions.

### PR DESCRIPTION
**Pull request by CharKeaney**
* bfd/ChangeLog: Added entry.
	* bfd/elfxx-riscv.c: Fixed RISC-V Zc* extensions version numbers.

### CODE REVIEW

File: gas/testsuite/ChangeLog.COREV [hunk 1]
```diff
@@ -1,3 +1,40 @@
...
+	* gas/testsuite/gas/riscv/csr-version-1p11.d: Likewise.
+	* gas/testsuite/gas/riscv/csr-version-1p11.l: Likewise.
+	* gas/testsuite/gas/riscv/csr-version-1p12.d: Likewise.
+	* gas/testsuite/gas/riscv/csr-version-1p12.l: Likewise.
+	* gas/testsuite/gas/riscv/csr-version-1p9p1.d: Likewise.
+	* gas/testsuite/gas/riscv/csr-version-1p9p1.l: Likewise.
+	* gas/testsuite/gas/riscv/csr-version-zcmt.d: Likewise.
+	* gas/testsuite/gas/riscv/csr-version-zcmt.l: Likewise.
+	* gas/testsuite/gas/riscv/csr.s: Likewise.
+
```
Comment (pietraferreira) [inline gas/testsuite/ChangeLog.COREV]: Extra line is here Charlie.

File: bfd/elfxx-riscv.c [hunk 2]
```diff
@@ -1288,13 +1288,13 @@ static struct riscv_supported_ext riscv_supported_std_z_ext[] =
...
-  {"zcmb",		ISA_SPEC_CLASS_DRAFT,		0, 7,  0 },
-  {"zcmp",		ISA_SPEC_CLASS_DRAFT,		0, 7,  0 },
-  {"zcmpe",		ISA_SPEC_CLASS_DRAFT,		0, 7,  0 },
-  {"zcmt",		ISA_SPEC_CLASS_DRAFT,		0, 7,  0 },
+  {"zca",		ISA_SPEC_CLASS_DRAFT,		0, 70,  4 },
+  {"zcb",		ISA_SPEC_CLASS_DRAFT,		0, 70,  4 },
+  {"zcf",		ISA_SPEC_CLASS_DRAFT,		0, 70,  4 },
+  {"zcmb",		ISA_SPEC_CLASS_DRAFT,		0, 70,  4 },
+  {"zcmp",		ISA_SPEC_CLASS_DRAFT,		0, 70,  4 },
+  {"zcmpe",		ISA_SPEC_CLASS_DRAFT,		0, 70, 4 },
```
Comment (pietraferreira) [inline bfd/elfxx-riscv.c:1296]: Missing space.

File: gas/testsuite/gas/riscv/csr-version-1p12.d [hunk 3]
```diff
@@ -571,3 +571,6 @@ Disassembly of section .text:
 [ 	]+[0-9a-f]+:[ 	]+c2159073[ 	]+csrw[ 	]+vtype,a1
 [ 	]+[0-9a-f]+:[ 	]+c2202573[ 	]+csrr[ 	]+a0,vlenb
 [ 	]+[0-9a-f]+:[ 	]+c2259073[ 	]+csrw[ 	]+vlenb,a1
+[ 	]+[0-9a-f]+:[ 	]+01702573[ 	]+csrr[ 	]+a0,jvt
+[ 	]+[0-9a-f]+:[ 	]+01759073[ 	]+csrw[ 	]+jvt,a1
+
```
Comment (pietraferreira) [inline gas/testsuite/gas/riscv/csr-version-1p12.d]: Extra line here.

File: bfd/ChangeLog [hunk 4]
```diff
@@ -1,3 +1,7 @@
+2022-08-22  Charlie Keaney  <charlie.keaney@embecosm.com>
+
+	* bfd/elfxx-riscv.c: Fixed COREV ZC version number.
```
Comment (simonpcook) [inline bfd/ChangeLog]: ZC is not a COREV extension

File: gas/testsuite/ChangeLog.COREV (hunk 1 above)
Comment (pietraferreira) [inline gas/testsuite/ChangeLog.COREV]: Extra line.

**Pull request merged**